public/js/site.js

```diff
@@ -1,3 +1,4 @@
+//TODO rewrite for materialize modal
```

Comment:
Currently I deleted the confirmation popups for the buttons in tabulators, because the bootstrap modals and materialize modals glitched together. If necessary, need to rewrite in a future issue, but this is mostly used by admins.